src/algorithms/spectral/tensorflowinputtempocnn.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright (C) 2006-2020  Music Technology Group - Universitat Pompeu Fabra
+ *
+ * This file is part of Essentia
+ *
+ * Essentia is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU Affero General Public License as published by the Free
+ * Software Foundation (FSF), either version 3 of the License, or (at your
+ * option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more
+ * details.
+ *
+ * You should have received a copy of the Affero GNU General Public License
+ * version 3 along with this program.  If not, see http://www.gnu.org/licenses/
+ */
+
+#include "tensorflowinputtempocnn.h"
+
+using namespace std;
+using namespace essentia;
+using namespace standard;
+
+const char* TensorflowInputTempoCNN::name = "TensorflowInputTempoCNN";
+const char* TensorflowInputTempoCNN::category = "Spectral";
+const char* TensorflowInputTempoCNN::description = DOC(
+  "This algorithm computes mel-bands with a particular parametrization specific to Hendrik Schreiber's TempoCNN based models.\n"
```

Comment:
Same here

Reply:
Simpler description: "This algorithm computes mel-bands input specific to TempoCNN-based models."  

or 

"This algorithm computes mel-bands specific to the input of TempoCNN-based models."

Is it still clear enough or should we still mention "parametrization "somewhere? (for those wondering why not to use MelBands instead). 

Similar simplification can be done other `TensorflowInput*` algorithms.